ui/react-components/components/Conflicts/Body/ConflictsBody.jsx

```diff
@@ -18,18 +18,18 @@ const ConflictsBody = props => {
         let tabHeads = [];
         let tabPanels = [];
         if (patientDoubleBookingConflicts) {
-            tabHeads.push(createTab(patientDoubleBookingConflicts, 'OVERLAPPING_CONFLICTS', 'Overlapping conflicts'));
+            tabHeads.push(createTab(patientDoubleBookingConflicts, 'OVERLAPPING_CONFLICTS', 'Overlapping conflicts', 'overlappingConflicts'));
             tabPanels.push(createTabPanel(patientDoubleBookingConflicts, OVERLAPPING_CONFLICTS_CONTENT));
         }
         if (serviceUnavailableConflicts) {
-            tabHeads.push(createTab(serviceUnavailableConflicts, 'NO_SERVICE_DATE_CONFLICTS', 'No-Service Date conflicts'));
+            tabHeads.push(createTab(serviceUnavailableConflicts, 'NO_SERVICE_DATE_CONFLICTS', 'No-Service Date conflicts', 'noServiceDateConflicts'));
             tabPanels.push(createTabPanel(serviceUnavailableConflicts, SERVICE_UNAVAILABLE_CONTENT));
         }
         return {tabHeads, tabPanels};
     };
 
-    const createTab = (conflicts, translationKey, defaultMessage) => {
-        return <Tab key={defaultMessage}>
+    const createTab = (conflicts, translationKey, defaultMessage, dataTestId) => {
```

Comment:
Why do we need different test id for same control? The content of the control should be enough for tests right?